scripts/listapranzo.js

```diff
@@ -85,6 +85,29 @@ module.exports = function (robot) {
     order.users[user.id] = dishes;
   };
 
+  var fuzzyMatch = function (dish, menuline) {
+    dish = dish.trim().toLowerCase().replace(" ", "");
+    var key = "";
+    for (var i = 0; i < dish.length; i++)
+      key = key + dish[i] + ".*"
```

Comment:
In ogni caso, opinione. Magari fare anche solo substring finding, però ordinato (scrivere "pe ro" matcha "penne rosa" e non "rosa penne") va bene. Alla fine si stanno scegliendo piatti, non credo cambi proprio così tanto.